packages/docs/docs/guides/8-glossary.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Glossary
+keywords:
+---
+
+# Ecosystem
+
+## Request Portal API
+
+The Request Portal is a software built on top of the Request Protocol that allows third party software to easily integrate Request, all the private keys to create requests and perform actions to them are stored by Request and can be managed through OAuth. The Request Portal API is an API built on top of the Request Portal.
+
+## Request Client
+
+The Request Client is a Javascript library made to interact directly with the Request Protocol. The Request Client connects to a Request Node.
+
+## Request Node
+
+Request Nodes are HTTP servers exposing an API used to allow Request Client to communicate with the Request Protocol. These servers abstract the complexity of IPFS and Ethereum used by the current implementation of the Request Protocol.
+
+## Request Protocol
+
+The Request Protocol is the underlying protocol to make Request work. The protocol stores all the request information and actions performed to them on a distributed ledger.
+
+# Request Protocol
+
+## Action
+
+An action is signed data added by a request's stakeholder into the Request Protocol that creates or updates the state of a request. The creation of a request is considered as an action and a request can be represented by a list of actions.
+
+## Balance
+
+When using a payment network, the balance is the current amount paid for a request. This balance is not the current balance of a specific account but is determined by the payment detection method of the payment network used.
+
+A request with no payment network provided doesn't have a balance.
+
+## Confirmed/Pending action
+
+The Protocol relies on other blockchain technologies to ensure data immutability. Most blockchain doesn't offer transaction instant finality. This means that when performing an action on the request, this action can't directly be confirmed as effective.
+
+As long as the action hasn't been persisted and is not confirmed, the action is marked as "pending".
+
+## Encryption provider
+
+An encryption provider is an object that allows to abstract the encryption of actions. It can be used to perform the action's encryption on a distant server.
```

Comment:
```suggestion
An encryption provider is an abstraction of the mechanism that handles encryption and decryption of a Request. Depending on use cases, it allows you to give your user full control, or handle some parts for them. 

It is not used for clear requests.
```

Reply:
"object" is weird here ;)
the part about the distant server feels confusing to me. My suggestion is just a hint of the direction I would take here, no need to use it as-is...